test/runtests.sh

```diff
@@ -2,27 +2,27 @@
 # USAGE: Run this file using `npm test` (must run from repository root)
```

Comment:
While we're at it, can we get rid of this and replace it in `package.json` with `test: mocha .test/*_test.js`?